chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/SvapController.java

```diff
@@ -16,10 +16,12 @@
 import gov.healthit.chpl.logging.Loggable;
 import gov.healthit.chpl.svap.domain.Svap;
 import gov.healthit.chpl.svap.manager.SvapManager;
-import io.swagger.annotations.Api;
-import io.swagger.annotations.ApiOperation;
+import gov.healthit.chpl.util.SwaggerSecurityRequirement;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import io.swagger.v3.oas.annotations.tags.Tag;
 
-@Api(value = "svaps")
+@Tag(name = "svaps", description = "Allows management of SVAPs.")
```

Comment:
Does this string show up in Swagger as a header or something? If so, maybe it would be a good place to spell out the acronym the first time, so we can use it in all of the places inside the controller. If it doesn't appear, no reason to do this
```suggestion
@Tag(name = "svaps", description = "Allows management of Standards Version Advancement Process (SVAPs).")
```